olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogClient.h

```diff
@@ -144,9 +142,8 @@ class DATASERVICE_READ_API CatalogClient final {
    * @deprecated Will be removed in 1.0
    */
   OLP_SDK_DEPRECATED("Will be removed in 1.0")
-  client::CancellationToken GetPartitions(
-      const PartitionsRequest& request,
-      const PartitionsResponseCallback& callback);
+  client::CancellationToken GetPartitions(PartitionsRequest request,
```

Comment:
This one should be removed before 1.0 so you dont need to update it. Same for all deprecated methods marked for 1.0

Reply:
Done